pkg/devfile/adapters/common/exec.go

```diff
@@ -39,9 +40,9 @@ func ExecuteCommand(client ExecClient, compInfo ComponentInfo, command []string,
 
 	if err != nil {
 		// It is safe to read from cmdOutput here, as the goroutines are guaranteed to have terminated at this point.
-		log.Errorf("\nUnable to exec command %v: \n%v", command, cmdOutput)
+		klog.V(4).Infof("ExecuteCommand returned an an err: %v. for command '%v'. output: %v", err, command, cmdOutput)
 
-		return err
+		return errors.Wrapf(err, "Unable to exec command %v: \n%v", command, cmdOutput)
```

Comment:
nitpick, but errors should not be capitalized or use newlines.

Reply:
See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
@cdrage : Done... nitpicking is always good, thanks for continuing to keep us honest on good error format 😃